src/main/java/com/java/dojo/recipes/entities/Comment.java

```diff
@@ -0,0 +1,26 @@
+package com.java.dojo.recipes.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Builder
+@EqualsAndHashCode(of = "id")
```

Comment:
Por qual motivo vc usou essa annotation do equals e hash code ?

Segundo o time do lombok:

Will soon be marked @Deprecated; use the @EqualsAndHashCode.Include annotation together with @EqualsAndHashCode(onlyExplicitlyIncluded = true).

Ela usando "of" vai ser deprecada em breve,,,

Reply:
Cara não manjo muito de lombok kkk

Reply:
Não sabia disso, vou corrigir dps